crates/bevy_text/src/text2d.rs

```diff
@@ -228,3 +230,27 @@ pub fn update_text2d_layout(
 pub fn scale_value(value: f32, factor: f32) -> f32 {
     value * factor
 }
+
+/// System calculating and inserting an [`Aabb`] component to entities with some
+/// [`TextLayoutInfo`] and [`Anchor`] components, and without a [`NoFrustumCulling`] component.
+///
+/// Used in system set [`VisibilitySystems::CalculateBounds`](bevy_render::view::VisibilitySystems::CalculateBounds).
+pub fn calculate_bounds_text2d(
+    mut commands: Commands,
+    text_to_update_aabb: Query<
+        (Entity, &TextLayoutInfo, &Anchor),
+        (
+            Or<(Without<Aabb>, Changed<TextLayoutInfo>)>,
+            Without<NoFrustumCulling>,
+        ),
+    >,
+) {
+    for (entity, layout_info, anchor) in &text_to_update_aabb {
+        commands.entity(entity).try_insert(Aabb {
+            center: (-anchor.as_vec() * layout_info.logical_size)
```

Comment:
Comments or tests here would be helpful to verify correctness :)

Reply:
Added some comments on the AABB computation and some tests inspired by those for the sprite AABBs :slightly_smiling_face: